### PR TITLE
in order for page to run locally, redis defaults to local database if no...

### PR DESCRIPTION
**Pull request by raorao**
... URL provided in ENV file

### CODE REVIEW

**Comment by nscricco** [comment]
:princess:

**Comment by nscricco** [comment]
Hey Zee, is there a better way to do this?

File: config/main.rb [hunk 1]
```diff
@@ -36,5 +36,10 @@
...
 require 'uri'
-uri = URI.parse(ENV['REDISTOGO_URL'])
-$redis = Redis.new(:host => uri.host, :port => uri.port, :password => uri.password)
+
+if ENV['REDISTOGO_URL']
+  uri = URI.parse(ENV['REDISTOGO_URL'])
+  $redis = Redis.new(:host => uri.host, :port => uri.port, :password => uri.password)
+else
+  $redis = Redis.new
+end
```
Comment (zspencer) [inline config/main.rb:45]: You can also use:

```
config :production do 
# production code here
end
config :development, :test do
# development/test codehere
end
```

Comment (zspencer) [inline config/main.rb:45]: ALternatively, you could require a REDIS_TO_GO_URL be placed in your .env and have no code branches!

**Pull request merged**